apps/web/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "description": "Event management software",
   "scripts": {
-    "dev": "next dev --turbo",
```

Comment:
Turbo is production ready for development. 
I don't see any reason to remove them

Reply:
@sreehari2003 revert this in next pr

Reply:
ya frontend was not working for me so i was trying to fix that , sorry about that